pwndbg/dbg_mod/gdb/__init__.py

```diff
@@ -1357,15 +1353,15 @@ def __getitem__(self, key: str | int) -> pwndbg.dbg_mod.Value:
             raise pwndbg.dbg_mod.Error(e)
 
 
-def _gdb_event_class_from_event_type(ty: pwndbg.dbg_mod.EventType) -> Any:
+def _gdb_event_registry_from_event_type(ty: EventType) -> gdb.EventRegistry[Any]:
     """
-    Returns the GDB event class that corresponds to the given event type.
+    Returns the GDB event registry that corresponds to the given event type.
     """
-    if ty == pwndbg.dbg_mod.EventType.EXIT:
+    if ty == EventType.EXIT:
```

Comment:
We should use match+case here? Also the events that are most used should be first cases etc